hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -369,6 +448,68 @@ protected void completeCompaction(HoodieCommitMetadata metadata, HoodieTable tab
     LOG.info("Compacted successfully on commit {}", compactionCommitTime);
   }
 
+  public void commitLogCompaction(String compactionInstantTime, HoodieWriteMetadata<O> compactionWriteMetadata, Option<HoodieTable> tableOpt,
```

Comment:
Does this need to be public? Let's make sure there are javadocs as well since the `completeLogCompaction` can also be used for commits based on the comments so it is not clear when a developer would want to call one or the other

Reply:
sure, will add java docs. 
I will ensure to follow up on access specifiers when putting out separate patches.